spec/unit/services/options_cleaner_spec.rb

```diff
@@ -0,0 +1,355 @@
+require 'spec_helper'
+
+describe Mongoid::History::OptionsCleaner do
+  class MyOptionsCleanerModel
```

Comment:
I do see that the rest of the code doesn't do that, so either change everything or just leave this alone, I am OK with it.